src/Compilers/VisualBasic/Portable/SymbolDisplay/SymbolDisplayVisitor.vb

```diff
@@ -181,10 +181,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Dim myCaseCorrectedNSName As String = symbol.Name
             Dim myCaseCorrectedParentNSName As String = String.Empty
 
-            If Not symbol.IsGlobalNamespace AndAlso myCaseCorrectedNSName.IsEmpty() Then
-                myCaseCorrectedNSName = StringConstants.NamedSymbolErrorName
```

Comment:
Yes, this led to the crash in `GetInfoForImmediateNamespaceMembers` since the length of the generated namespace did not match the actual length.